### PR TITLE
optimize: add the default console and security config on application.examle.yml

### LINKED ISSUE #6725: 建议在application.example.yml文件中加上console.*和security.*配置
## Why you need it?
当我下载seata release v2.0后，通常我是把application.yml改为application.yml.bak,  application.example.yml改为application.yml。
但是当我把一些初始配置配置好后启动报错：
> Error creating bean with name 'customUserDetailsServiceImpl': Injection of autowired dependencies failed; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'console.user.username' 

经过上网查询后了解到需要再配置文件里补充上以下内容：
```yaml

console: 
  user:
    username: seata
    password: seata

seata:
  security:
    secretKey: "seata"
    tokenValidityInMilliseconds: 1000000000

```

## How it could be?
希望在`application.example.yml`配置文件中默认添加上这些内容, 谢谢 !

### PR DESCRIPTION
**Pull request by hmj776521114**
### Ⅰ. Describe what this PR did
add the default console and security config on application.examle.yml(#6725)

### Ⅱ. Does this pull request fix one issue?

fixes #6725 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
it donnot need

### Ⅳ. Describe how to verify it
unzip the seata-server.zip, and  rename `application.yml` to `application.yml.bak`, `application.example.yml` to `application.yml` on `seata-server`. then execute the `seata-server.sh` shell script, check whether startup successfully.

### CODE REVIEW

File: server/src/main/resources/application.example.yml
Comment (funky-eyes) [inline server/src/main/resources/application.example.yml:1]: https://github.com/apache/incubator-seata/blob/2.x/server/src/main/resources/application.raft.example.yml
应该也要进行补充
It should also be supplemented.

**Comment by funky-eyes** [comment]
changes 下的2.x.md 中英两个版本都要登记下pr信息和作者信息。
Please register pull request information and author details in both the English and Chinese versions of the 2.x.md file located in the changes folder.

File: server/src/main/resources/application.example.yml
Comment (hmj776521114) [reply server/src/main/resources/application.example.yml:1]: 好的  已补充

**Comment by hmj776521114** [comment]
> changes 下的2.x.md 中英两个版本都要登记下pr信息和作者信息。 Please register pull request information and author details in both the English and Chinese versions of the 2.x.md file located in the changes folder.

okay, all done.

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**